libs/sdk-ui-kit/src/Header/generateHeaderStaticHelpMenuItems.ts

```diff
@@ -12,11 +12,17 @@ const addUTMParameters = (baseUrl: string, utmContent: TUTMContent): string =>
 /**
  * @internal
  */
-export function generateHeaderStaticHelpMenuItems(): IHeaderMenuItem[] {
-    const universityUrl = "https://university.gooddata.com/page/gooddatacn";
-    const communityUrl = "https://community.gooddata.com";
-    const slackUrl = "https://www.gooddata.com/slack/";
-    const documentationUrl = "https://www.gooddata.com/developers/cloud-native/doc/";
+export function generateHeaderStaticHelpMenuItems(
+    documentationLink?: string,
+    communityLink?: string,
+    universityLink?: string,
+    slackLink?: string,
+): IHeaderMenuItem[] {
```

Comment:
yes, we can. But It should be the same with other generate menu item functions like https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-kit/src/Header/generateHeaderAccountMenuItems.ts#L16
or https://github.com/gooddata/gooddata-ui-sdk/blob/9d6cb4fa2a6f0c71c0c24ace04eca17ae6562134/libs/sdk-ui-kit/src/Header/generateHeaderHelpMenuItems.ts#L20

Reply:
Actually, I don't like call the function like: 
eg: ` generateHeaderStaticHelpMenuItems(null, null, null, "SlackUrl");` :).

Reply:
IMO, we should follow the way [generateHeaderHelpMenuItems](https://github.com/gooddata/gooddata-ui-sdk/blob/9d6cb4fa2a6f0c71c0c24ace04eca17ae6562134/libs/sdk-ui-kit/src/Header/generateHeaderHelpMenuItems.ts#L20) is defined (having multiple "flat" parameters), for the sake of consistency.

In addition, deliberately specifying `null` to a parameter when calling `generateHeaderStaticHelpMenuItems` makes it clear that the parameter should be null. Whereas, using object as a single parameter is somewhat more error-prone, as one can omit the corresponding field in the object if it doesn't hold any value, but it might also mean he simply forgets to set value for the field...

However, for the sake of consistency, should we change `Link` in the parameters' names to `Url`? like the way parameters of `generateHeaderHelpMenuItems` are named:

- `documentationLink` → `documentationUrl`
- `communityLink` → `communityUrl`
- `universityLink` → `universityUrl`
- `slackLink` → `slackUrl`

Reply:
Ok, I have updated this. Thanks!